src/backend/src/modules/pia-intake/dto/create-pia-intake.dto.ts

```diff
@@ -7,10 +7,39 @@ import {
   IsString,
 } from '@nestjs/class-validator';
 import { ApiProperty } from '@nestjs/swagger';
-
-import { piaIntakeEntityMock } from 'test/util/mocks/data/pia-intake.mock';
```

Comment:
for whoever reviewing - This is the line that was the culprit. Fetching mock data from the `test` folder was breaking the build.